ooni/errors.py

```diff
@@ -148,6 +149,9 @@ def failureToString(failure):
     elif isinstance(failure.value, ProcessDone):
         string = 'process_done'
 
+    elif isinstance(failure.value, ConnectionDone):
+        string = 'connection_done'
```

Comment:
Ok, i'll add it to the specification.

Maybe it's not a good idea to give a more detailed error message as we don't have enough information on why the error happen.